### ISSUE TITLE
Date range form hack causes a11y test error  (severity 6)

### ISSUE THREAD

**Issue by jvine** [sul-dlss/exhibits#1616]
On pages with a date range facet, there's a Blacklight workaround in the range form that is reported as an accessibility error. 

```
<!-- we need to include a dummy search_field parameter if none exists,
           to trick blacklight into displaying actual search results instead
           of home page. Not a great solution, but easiest for now. -->
        <input type="hidden" name="search_field" id="search_field" value="dummy_range" data-la-initdispnone="true">
```

The issue is that `id="search_field"` is a duplicate ID of the main search field.

I don't think this has a serious effect IRL, since the duplicate input is hidden, but it does get reported as a severity 6 (out of 10) error. (Note that SearchWorks has the same comment embedded in its HTML, but does not have the same error in its tests: there's no duplicate ID.)

**Comment by mejackreed** [comment]
As far as I can tell this issue no longer presents. @jvine can you please confirm?

**Comment by jvine** [comment]
Re-tested with AMP and the error is no longer reported. Closing.
